configs/neox_arguments.md

```diff
@@ -1845,11 +1847,3 @@ Args for deepspeed runner (deepspeed.launcher.runner).
 
     Adds a `--comment` to the DeepSpeed launch command. In DeeperSpeed this is passed on to the SlurmLauncher as well. Sometime necessary for cluster rules, or so I've heard.
 
-
-
-- **no_ssh_check**: bool
```

Comment:
There was no mention of this being removed in the PR. Just want to check that it's deliberate / record the motivation.

Reply:
It was done automatically in https://github.com/EleutherAI/gpt-neox/pull/824/commits/577126cf7c8532c7c8c29b04e173603987a37410 looks like. I'll take a look later as to why.

Reply:
This will be resolved by https://github.com/EleutherAI/gpt-neox/pull/831